spec/praxis/endpoint_definition_spec.rb

```diff
@@ -12,7 +12,7 @@
 
   its(:prefix) { should eq('/people') }
 
-  its(:actions) { should have(2).items }
+  its(:actions) { should have(4).items }  # Two real actions, and twp post version of the GET
```

Comment:
Typo in comment, should be "Two real actions, and two post version of the GET"

Reply:
👍